### PR TITLE
Large screen device

### PR DESCRIPTION
**Pull request by aishatpopoola**
This PR does the following:
. made changes in the header section for the large-screen device
. made changes in the home section for large-screen device
. made changes in the services section for the large-screen device.
. made changes in the portfolio section for the large-screen device.
. add the swiper feature to the portfolio section for the large-screen device
. made changes in the contact section for the large-screen device

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,6 +1,8 @@
+/* @import './swiper-bundle.min.css'; */
```
Comment (adejam) [inline css/index.css]: Please let's remove commented codes

File: index.html [hunk 2]
```diff
@@ -739,6 +741,27 @@ <h1 class="footer_title">Aishat</h1>
...
+
+            },
+            pagination: {
+                e1: '.swiper-pagination',
+                clickable: true,
+            },
+
+        });
+
+    </script> -->
```
Comment (adejam) [inline index.html]: Remove commented codes

**Review by adejam — APPROVED** [review]
#LGTM

Please remove commented codes and merge.
Wishing you the best on your next milestone

File: css/index.css (hunk 1 above)
Comment (aishatpopoola) [reply css/index.css]: oh i forgot, thanks

File: index.html (hunk 2 above)
Comment (aishatpopoola) [reply index.html]: ok

**Comment by aishatpopoola** [comment]
I removed all the comments as mentioned in the preview

**Pull request merged**